### DISCUSSION TITLE
Exclude Words in Generation DOTO

### DISCUSSION THREAD

**Discussion by olsza** [Laravel-Lang/common#56] [Q&A] [answered]
Regarding the topic that was violated in #1373 , that is, when generating the todo.md file, to exclude words that are written in the same language in the same language as in the original language.

My proposal is on my account in branch future/excludeWords, i.e. https://github.com/olsza/Laravel-lang/tree/future/excludeWords

**Comment by caouecs** [comment, marked as answer]
Can you do a pull request to test your work ?

**Reply by olsza** [reply]
> Can you do a pull request to test your work ?

hsy, I added pull request #1378

**Reply by andrey-helldar** [reply]
@olsza, I did a code review, there are comments.

**Reply by olsza** [reply]
Hay @andrey-helldar , thanks for the code review but I don't see any comments :(
you can tell me where and how to see it, because I'm looking, but I can't find it, and I'm not hiding that before I saw this message, I added a new push and maybe that's why I can't see: /
And as I once mentioned, I'm just getting started in open sourse on gitHub

**Reply by andrey-helldar** [reply]
https://github.com/Laravel-Lang/lang/pull/1378
![image](https://user-images.githubusercontent.com/10347617/94819135-94a42680-0407-11eb-9428-c483b76a7e68.png)

**Reply by olsza** [reply]
Although I still can't read these comments anywhere, natet checked in the settings and I have marked show the comment in the code in github, I do not see them, maybe some additional settings I do not have ://
but thanks for the screenshot of the comments, at least I can read from them. :)
I'll fix it and send it again tomorrow.
ps. It is possible that these comments are somehow not approved or are only visible to you ??
ps2. and you see my 2 test comments ??

**Reply by olsza** [reply]
hay @andrey-helldar , I was able to connect to the computer, where I have everything set up with this project and corrected after 'code review' but I only corrected what I saw on the print screen, if there was something else, please let me know.
Additionally, in this branch "additional information" is added, i.e. how many translations are missing (I know, I should probably do it in a separate branch), because maybe this addition may not get a positive rating:/
but if necessary, I will do everything as it should, if you reject

**Reply by andrey-helldar** [reply]
Yes, I see test comments. The code review should be visible. I'll check it soon.

**Reply by andrey-helldar** [reply]
It's strange. All comments for the code review are visible even without logging into the github.

![image](https://user-images.githubusercontent.com/10347617/94850624-dea10280-042f-11eb-93fe-aa69ddc9f991.png)

![image](https://user-images.githubusercontent.com/10347617/94850640-e2cd2000-042f-11eb-8f4d-412ccad7b5f3.png)

**Reply by olsza** [reply]
comments started popping up :) and I already gave a "push" with corrections: D
thanks @andrey-helldar

**Comment by andrey-helldar** [comment]
Sorry, I never got to write about this. A lot of work.

**Reply by olsza** [reply]
ok, no problem, it's not a priority

**Comment by andrey-helldar** [comment]
@Laravel-Lang/laravel-lang, @olsza proposes to create files with phrases exclusion for correct generation of todo.md file

There are [two opinions](https://github.com/Laravel-Lang/lang/pull/1378/files/2b1a71b543a6aa19295b8d6d2d92737de4dc5e7d..956668de36c45e3f08abbb56c5b8dc79566b40a5):

![image](https://user-images.githubusercontent.com/10347617/94899015-2f017a00-049b-11eb-9ea9-1feecb6e11cc.png)

What do you say? What are your thoughts on this?

**Reply by andrey-helldar** [reply]
The word "Email" can also be translated into Russian as "Электронный адрес", "Адрес электронной почты", "Электронная почта", "Емейл".
Can also write both "Email" and "E-mail".
But we more often use the English name "Email" without a hyphen.

**Reply by andrey-helldar** [reply]
I suggest using the option suggested by @olsza, but you need the @Laravel-Lang/laravel-lang's opinion on this matter.

**Comment by caouecs** [comment]
I have a lot of messages to read this week-end... luckily it's raining ;)

**Reply by andrey-helldar** [reply]
😁

**Comment by luisprmat** [comment]
I think it's a great idea add exclusion files for language only if necessary. I'm agree!!

**Reply by mohamedsabil83** [reply]
Me too :+1:

**Reply by luisprmat** [reply]
Feature implemented!!

**Reply by mohamedsabil83** [reply]
Yep, I know. My reply means that I confirm that's **a great idea**.

**Comment by luisprmat** [comment]
Thanks @olsza, I had contributed with the documentation of this new feature. [View](https://github.com/Laravel-Lang/lang/blob/master/CONTRIBUTING.md#what-should-i-do-if-there-is-a-tag-whose-translation-is-the-same-as-in-english)
